app/configurator/components/chart-options-selector.tsx

```diff
@@ -1130,8 +1124,9 @@ const ChartFieldAnimation = ({ field }: { field: AnimationField }) => {
                 />
               </Flex>
             </Box>
-            <Box sx={{ display: "flex", alignItems: "center", mt: 5 }}>
+            <Box display="flex" alignItems="center" mt={5} gap="0.5rem">
```

Comment:
No good reason. I would think that it would in the end create less objects, but it will not change anything I guess. I think I find it cleaner to use the props when they are there.